### PR TITLE
Add further html pages

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bookmarks</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <header><h1>Bookmarks</h1></header>
```
Comment (badabam) [inline bookmarks.html]: No need for a `header` element, if there is only one child element. Just style the `h1` element directly.

File: bookmarks.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem
+          ipsum dolor sit amet.
+        </details>
+        <ul role="list">
+          <li>tag 1</li>
+          <li>tag 2</li>
+          <li>tag 3</li>
+          <li>tag 4</li>
+        </ul>
+      </section>
```
Comment (badabam) [inline bookmarks.html:34]: Nice!

File: bookmarks.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    </main>
+    <footer>
```
Comment (badabam) [inline bookmarks.html]: No need for the `footer` element, if there is only one child – like in your case the `nav` element. Just use the `nav`.

File: bookmarks.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      <nav>
+        <ul role="list">
+          <li>
+            <a href="./index.html"
+              ><img src="./images/home.png" alt="" width="40px" height="40px"
```
Comment (badabam) [inline bookmarks.html]: Remember: no units on `width` and `height` of an `img`.
```suggestion
              ><img src="./images/home.png" alt="" width="40" height="40"
```

Comment (badabam) [reply bookmarks.html]: Also: if possible, an SVG would be preferable to also look great on high resolution displays.

**Review by badabam — CHANGES_REQUESTED** [review]
Very nice! Please remove the obsolete elements and update your styling, if needed. Great work!

File: bookmarks.html (hunk 3 above)
Comment (ahohnsen) [reply bookmarks.html]: Good to know, thx!

File: bookmarks.html (hunk 4 above)
Comment (ahohnsen) [reply bookmarks.html]: Ok, now I got it :)

**Pull request merged**